tests/functional/cli/version.sh

```diff
@@ -2,10 +2,12 @@
 
 # SPDX-License-Identifier: Apache-2.0
 #
-# Copyright (C) 2021 Micron Technology, Inc. All rights reserved.
+# Copyright (C) 2021-2022 Micron Technology, Inc. All rights reserved.
 
 . common.subr
 
+trap remove_home EXIT
```

Comment:
A function dedicated to trap cleanup (and documented as such) would be good for several reasons:
- in the future, we can add to cleanup function instead of re-editing every use of `trap`
- might avoid using `cmd` since it does things that trap might not want (replicate checks for FAKE or whatever else `cmd` does)

Reply:
Do any of these smokes use background tasks?  If so, more trap cleanup is needed.

Reply:
> Do any of these smokes use background tasks? If so, more trap cleanup is needed.

I am not aware of any.

> A function dedicated to trap cleanup (and documented as such) would be good

Makes sense. I will change the name of the current function to `cleanup` and remove the usage of `cmd` within it. In addition, I will add a comment to the function.

Reply:
There's black magic involved in successfully killing background jobs.  See:

https://stackoverflow.com/questions/360201/how-do-i-kill-background-processes-jobs-when-my-shell-script-exits

look for this:
```
trap 'trap - SIGTERM && kill 0' SIGINT SIGTERM EXIT
```

Reply:
That's good to know. I might document this SO answer in common.subr in the event we ever need it.

Reply:
I could not get that to work properly.